hello.py

```diff
@@ -7,14 +7,17 @@
 
 @app.route('/')
 def hello_world():
+    """Return a simple greeting message."""
```

Comment:
i want this doc to be more expressive like what is the input and the output and the params